spec/features/create_account_spec.rb

```diff
@@ -23,6 +23,14 @@
 
       log_out_from_oidc_sp
     end
+
+    it 'creates new IAL2 account with SMS option for 2FA' do
```

Comment:
It looks like this spec is wired up to run in prod. Since we aren't accepting the test files there won't this fail there?